voice/fi/ttsconfig.p

```diff
@@ -86,27 +86,26 @@ string('follow.ogg', 'Seuraa tietä ').
 
 % ARRIVE
 string('and_arrive_destination.ogg', 'ja olet perillä ').
-string('reached_destination.ogg', 'olet perillä ').
+string('reached_destination.ogg', 'olet perillä, ').
```

Comment:
Is this change really necessary?

Reply:
It's for adding a little pause before the destination name which can come after this. Without the comma it's just continuous: "olet perillä koti" ("koti" means home). With the comma, there's a slight pause: "olet perillä, koti". If there's another way to add that pause, then I'm willing to learn that.

Reply:
And the whole phrase just sounds more humane with that pause. Without the pause it sounds like a machine -- which of course it is :)